### DISCUSSION TITLE
when `/content`  file name contains chinese，will generate same path route

### DISCUSSION THREAD

**Discussion by baixiaoyu2997** [nuxt/content#1891] [Q&A] [answered]
if i have two file ,like `/node文档.md` and `/node.md`,  they `_path` are all `/node` .

how to resolve this issue?

**Comment by nobkd** [comment, marked as answer]
See https://github.com/nuxt/content/issues/1383#issuecomment-1196237686 for context

**Reply by baixiaoyu2997** [reply]
thanks!!!
